blocks/api/post.pegjs

```diff
@@ -45,13 +45,13 @@ WP_Block_Type
 WP_Block_Attribute_List
   = as:(_+ attr:WP_Block_Attribute { return attr })*
   { return as.reduce( function( attrs, pair ) {
-    attrs[ pair.name ] = [ pair.value ];
+    attrs[ pair.name ] = pair.value;
     return attrs;
   }, {} ) }
 
 WP_Block_Attribute
   = name:WP_Block_Attribute_Name ":" value:WP_Block_Attribute_Value
-  { return [ name, value ] }
+  { return { name, value }; }
```

Comment:
There's an issue where this file isn't transpiled, and UglifyJs chokes on this line. You can see the error when running `npm run build`. The workaround was just to use ES5 in the file, otherwise we need to investigate how to apply Babel to the file.